lib/msal-node/test/client/ManagedIdentitySources/AppService.spec.ts

```diff
@@ -176,6 +178,61 @@ describe("Acquires a token successfully via an App Service Managed Identity", ()
         });
     });
 
+    describe("Miscellaneous", () => {
+        let managedIdentityApplication: ManagedIdentityApplication;
+        beforeEach(() => {
+            managedIdentityApplication = new ManagedIdentityApplication(
```

Comment:
What about client capabilities?

Reply:
Do you just want me to check that "xms_cc" is sent in the network request when client capabilities are provided in the config? I thought that might be too to warrant a unit test, but I can add that if you'd like.

Reply:
Of course, we should aim for max code coverage. It's also how a CX would write code - they would declare cp1 capability.

Reply:
done